packages/@aws-cdk/aws-codepipeline-actions/test/fake-build-action.ts

```diff
@@ -0,0 +1,55 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import * as iam from '@aws-cdk/aws-iam';
+import { IResource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+
+export interface FakeBuildActionProps extends codepipeline.CommonActionProps {
```

Comment:
I guess it might, but it felt like something I didn't want to bother with at all. Only thing I want is the Pipeline not to yell at me here, I don't really want to bother coming up with sane props for a `CodeBuildAction` (such as a `Project` etc).